### PR TITLE
Update quickstart with Sink example of Kafka Connector 3.1

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

**Comment by avsej** [comment]
@ingenthr @1tylermitchell it is GA release with experimental feature. We called it 3.1 in our release plan (I haven't made tag yet). So question, should it be another directory in the docs, or it can leave under 3.0?

**Comment by 1tylermitchell** [comment]
Make it a new, 3.1, version to be pegged to 4.6 server docs only.

**Comment by mattcarabine** [comment]
Closing + Re-opening for CV

**Comment by mattcarabine** [comment]
Realised the link wasn't posted above: http://ec2-54-197-14-204.compute-1.amazonaws.com:8080/job/Server/job/pull-requests/job/build-pr/84/parsed_console/

**Comment by amarantha-k** [comment]
Please create a new directory for 3.1 and include the Kafka 3.1 files in there.

File: content/connectors/kafka-3.0/quickstart.dita [hunk 1]
```diff
@@ -146,5 +146,22 @@ use_snapshots=false
...
+            into new empty bucket. We assume that a bucket named <codeph>receiver</codeph> exists on the server. Our configuration will look like this:</p>
+    <codeblock outputclass="language-text"><![CDATA[name=test-sink-couchbase
+connector.class=com.couchbase.connect.kafka.CouchbaseSinkConnector
+tasks.max=1
+topics=test-default
+connection.cluster_address=127.0.0.1
+connection.bucket=receiver]]></codeblock>
+            <p>Now if you run this job, it will report some diagnostic information about processing data to standard
+            output, and soon <codeph>receiver</codeph> bucket documents number will match the source
+            <codeph>travel-sample</codeph>:
```
Comment (1tylermitchell) [inline content/connectors/kafka-3.0/quickstart.dita]: missing closing p tag

**Comment by mattcarabine** [comment]
Looks like all of the links 404.... has the ditamap been updated to point to the new pages?

**Pull request merged**